tests/shutdown/grub_set_bootargs.pm

```diff
@@ -16,16 +16,32 @@ use testapi;
 
 sub run() {
     select_console('root-console');
-    script_run "source /etc/default/grub";
-    script_run 'new_cmdline=`echo $GRUB_CMDLINE_LINUX_DEFAULT | sed \'s/\(^\| \)quiet\($\| \)/ /\'`';
-    if (get_var("ENABLE_FIPS")) {
-        script_run 'new_cmdline="$new_cmdline fips=1"';
-        if (get_var("ENCRYPT") && !get_var("FULL_LVM_ENCRYPT")) {
-            script_run 'new_cmdline="$new_cmdline boot=$(df /boot | tail -1 | cut -d" " -f1)"';
+    my @cmds;
+    push @cmds, "source /etc/default/grub";
+    push @cmds, 'new_cmdline=`echo $GRUB_CMDLINE_LINUX_DEFAULT | sed \'s/\(^\| \)quiet\($\| \)/ /\'`';
+    push @cmds, 'sed -i "s#GRUB_CMDLINE_LINUX_DEFAULT=\"[^\"]*\"#GRUB_CMDLINE_LINUX_DEFAULT=\"$new_cmdline\"#" /etc/default/grub';
+    push @cmds, 'grub2-mkconfig -o /boot/grub2/grub.cfg';
+
+    # Slow type for 12-SP2 aarch64 image creatation test to try to avoid filling up the key event queue
+    if (check_var('ARCH', 'aarch64') && check_var('VERSION', '12-SP2')) {
+        for my $cmd (@cmds) {
+            type_string $cmd . "\n", wait_screen_change => 1;
```

Comment:
Rebased, and I think it is better to keep aarch64 specific based on above I mentioned. Also to hack script_run with slow type option I would put it later even already a dirty-hack on my side. This only a preparation work for aarch64 migration test, so I don't want to spend long time on it. And following are verified runs on x86_64:
normal type: http://147.2.207.208/tests/544
slow type: http://147.2.207.208/tests/545